components/ProfileDetailEl.js

```diff
@@ -0,0 +1,18 @@
+import Image from './Image';
+
+export default function ProfileDetail({ detail }) {
+  const { description, images } = detail.fields;
+  const { url } = images[0].fields.file;
+
+  return (
+    <article className="profile-detail-el">
+      <Image
+        url={url}
+        classNameWrapper="profile-detail-el__w-img"
+        className="profile-detail-el__img"
+        alternativeText=""
```

Comment:
You have to add the role attribute with value decorative as described in ticket. The Image component already receive the role attribute.

Reply:
Thanks man, its fixed